✨ Add filter functionality to table editor

Nuhvi opened the pull request:
Add real-time filter functionality to the table editor, similar to the one used on [Material-UI Table component]( https://material-ui.com/components/tables/#material-table)

Nuhvi committed:
✨ Add filter functionality to table editor

imhta reviewed (changes requested):
### minor changes required
Everything looking good, couple things to do
- [x] add option  to enable and disable search field
- [x] move filter function to util 

----
**Pon Muthu Selvam**

Nuhvi committed:
✨ Add Searchable option to the LiveTableEditor

Nuhvi committed:
🚜 Move filterRows function to utils

Export utils function from an index file and separate each funcion.

Show "no records to display" if filtered data is empty.

imhta reviewed (approved):
# 👍
